frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -240,21 +240,50 @@ export const getReportBugLink = (cv: ClusterVersionKind): { label: string; href:
     return null;
   }
 
-  // Show a Bugzilla link for prerelease versions and a support case link for supported versions.
   const { major, minor, prerelease } = parsed;
-  const bugzillaVersion = major === 4 && minor <= 3 ? `${major}.${minor}.0` : `${major}.${minor}`;
-  const environment = encodeURIComponent(`Version: ${version}
-Cluster ID: ${cv.spec.clusterID}
-Browser: ${window.navigator.userAgent}
-`);
+  let productName;
+  switch (window.SERVER_FLAGS.branding) {
+    case 'openshift':
+    case 'ocp':
+      productName = 'OpenShift%20Container%20Platform';
+      break;
+    case 'online':
+      productName = 'OpenShift%20Online';
+      break;
+    case 'dedicated':
+      productName = 'OpenShift%20Dedicated';
+      break;
+    case 'azure':
+      productName = 'OpenShift%20Managed%20(Azure)';
+      break;
+    default:
+      productName = 'OKD';
+  }
+  const bugLabel = i18next.t('public~Report bug to Red Hat');
+
+  // Show an OpenShift Container Platform Bugzilla link for OKD
+  // until the OKD product components are updated.
+  if (productName === 'OKD') {
+    const bugzillaVersion = major === 4 && minor <= 3 ? `${major}.${minor}.0` : `${major}.${minor}`;
+    const environment = encodeURIComponent(`Version: ${version}
+    Cluster ID: ${cv.spec.clusterID}
+    Browser: ${window.navigator.userAgent}
+    `);
+    return {
+      label: bugLabel,
+      href: `https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform&version=${bugzillaVersion}&cf_environment=${environment}`,
+    };
+  }
+
+  // Show a support case link for supported versions and a Jira link for prerelease versions.
   return _.isEmpty(prerelease)
     ? {
         label: i18next.t('public~Open support case with Red Hat'),
-        href: `https://access.redhat.com/support/cases/#/case/new?product=OpenShift%20Container%20Platform&version=${major}.${minor}&clusterId=${cv.spec.clusterID}`,
+        href: `https://access.redhat.com/support/cases/#/case/new?product=${productName}&version=${major}.${minor}&clusterId=${cv.spec.clusterID}`,
       }
     : {
-        label: i18next.t('public~Report bug to Red Hat'),
-        href: `https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform&version=${bugzillaVersion}&cf_environment=${environment}`,
+        label: bugLabel,
+        href: `https://issues.redhat.com/secure/CreateIssue.jspa?pid=12332330`,
```

Comment:
@spadgett suggested we may be able to prefill the OpenShift version and perhaps the cluster ID in the Jira issue, but we’d have to investigate if we can set that through a query parameter.  I do not appear to have the ability within Jira to do so however.

Reply:
@rhamilto Looks like it is possible: https://confluence.atlassian.com/jirakb/how-to-create-issues-using-direct-html-links-in-jira-server-159474.html

Reply:
Turns out it isn't.  :(  See https://github.com/openshift/console/pull/11874/files#diff-ab468773bf6bdc82888f5abe2318066860f9e030113098584bba00ffbace2463R288